File: JiWonLee/baekjoon/2839.swift
var n = Int(readLine()!)!
var count = 0

while n > 0 {
    if n % 5 == 0 {
        n -= 5
        count += 1
    } else if n % 3 == 0 {
        n -= 3
        count += 1
    } else if n > 5 {
        n -= 5
        count += 1
    } else if n > 3 {
        n -= 3
        count += 1
    } else {
        count = -1
        break
    }
}
print(count)

File: Seah-An/baekjoon/greedy/2839.py
import sys

input = sys.stdin.readline

N = int(input())

cnt = 0

while N >= 0:

    if N % 5 == 0:
        cnt += (N // 5)
        print(cnt)
        break
    N -= 3
    cnt += 1
else:
    print(-1)

